packages/blocks/src/api/raw-handling/utils.js

```diff
@@ -113,14 +113,49 @@ export function getBlockContentSchema( context ) {
 
 /**
  * Checks whether HTML can be considered plain text. That is, it does not contain
- * any elements that are not line breaks.
+ * any elements that are not line breaks, or it only contains a single non-semantic
+ * wrapper element (span) with no semantic child elements.
  *
  * @param {string} HTML The HTML to check.
  *
  * @return {boolean} Whether the HTML can be considered plain text.
  */
 export function isPlain( HTML ) {
-	return ! /<(?!br[ />])/i.test( HTML );
+	// Existing check: no tags except <br>
+	if ( ! /<(?!br[ />])/i.test( HTML ) ) {
+		return true;
+	}
+
+	// New check: single wrapper element with no semantic children
+	try {
+		const doc = document.implementation.createHTMLDocument( '' );
+		doc.body.innerHTML = HTML;
+
+		// Check if there's exactly one top-level element
+		const childElements = Array.from( doc.body.children );
+		if ( childElements.length !== 1 ) {
+			return false;
+		}
+
+		const wrapper = childElements[ 0 ];
```

Comment:
Since we don't do any advanced array manipulation, we could actually use `document.body.children` collection and avoid an extra variable + `Array.from()`:

```suggestion
		if ( doc.body.children.length !== 1 ) {
			return false;
		}

		const wrapper = doc.body.children.item(0);
```